packages/init/templates/package.json.hbs

```diff
@@ -53,7 +53,7 @@
         "style": "import-sort-style-4catalyzer"
       },
       {{else}}
-      ".ts, .tsx": {
+      ".js, .ts, .tsx": {
```

Comment:
alternatively we could always render the block above

Reply:
meh this is fine. eventually we'll only have one parser and the filename will be passed to import-sort so it doesn't need to do this at all